packages/salesforcedx-sobjects-faux-generator/src/generator/fauxClassGenerator.ts

```diff
@@ -81,8 +91,17 @@ export class FauxClassGenerator {
     ['complexvalue', 'Object']
   ]);
 
-  private static fieldName(decl: string): string {
-    return decl.substr(decl.indexOf(' ') + 1);
+  private static fieldDeclToString(decl: FieldDeclaration): string {
+    return `${FauxClassGenerator.commentToString(decl.comment)}${INDENT}${
+      decl.modifier
+    } ${decl.type} ${decl.name};`;
+  }
+
+  private static commentToString(comment?: string): string {
+    // for some reasons if the comment is on a single line the help context shows the last '*/'
+    return comment
+      ? `${INDENT}/* ${comment.replace(/\*\//g, '')}${EOL}${INDENT}*/${EOL}`
```

Comment:
Is the purpose of the `replace` to sanitize the inlineHelpText if it has any comment block delimiters? If so should we add `/*` as well?